server/notebooks/templates/notebook_uploaded_files.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
```

Comment:
I think we can use the existing `notebook.html` template here with a few adjustments-- see my branch for details.

(but don't worry about making this change, I can do it when I merge your changes in)